code/ui/manager/package.json

```diff
@@ -59,6 +59,7 @@
     "@storybook/components": "workspace:*",
     "@storybook/core-events": "workspace:*",
     "@storybook/global": "^5.0.0",
+    "@storybook/icons": "^1.2.0",
```

Comment:
This needs to be a dependency not a devDependency, otherwise it will get prebundled & therefore duplicated

Reply:
Ok I fixed it.

Reply:
Based on our conversations, I moved the import of `@storybook/icons` as a `devDependency` on the manager only. Every other imports are set as `dependencies`.